packages/rx_bloc_list/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: rx_bloc_list
 description: The rx_bloc_list package facilitates implementing infinity scroll and pull-to-refresh features with minimal setup. This package is meant to be used along with RxBloc ecosystem.
-version: 4.0.0
```

Comment:
When upgrading dev dependencies we do not need to bump the version of the package